frontend/src/scenes/session-recordings/player/inspector/components/PlayerInspectorListItem.tsx

```diff
@@ -175,6 +179,10 @@ export function PlayerInspectorListItem({
                     <div className="flex items-start p-2 text-xs">
                         {item.offline ? 'Browser went offline' : 'Browser returned online'}
                     </div>
+                ) : item.type === 'browser-visibility' ? (
+                    <div className="flex items-start p-2 text-xs">
+                        <span>Window became</span> {item.status}
```

Comment:
This looks like it's not rendering the space in the screenshot? Perhaps it's old, but if not is there a reason we need the extra span? Otherwise you can probably just add the space manually
```suggestion
                        <span>Window became</span>{' '}{item.status}
```

Reply:
weird prettier remove the space until I removed the span 🤷 obey prettier :)